### DISCUSSION TITLE
Length of category

### DISCUSSION THREAD

**Discussion by ghost** [moneymanagerex/moneymanagerex#4834] [Q&A] [answered]
i'm using  win64 version in french.

Since 1.5.16, Category combobox is really short. 
When you use subcategory, you can't see the main info on the line.
![1](https://user-images.githubusercontent.com/81561095/179520060-a2adead7-0669-4117-b7b2-050c80f1a5a9.png)

**Comment by ghost** [comment]
Archived category are also displayed :)

**Comment by tactilis** [comment, marked as answer]
1. You can press the Enter key in the empty (blank) Category field and the Organize Categories dialog will be displayed.
or
2. You can start typing the category name that you want, for example C64, and the list will be filtered

**Reply by ghost** [reply]
Perfect ! didn't know that !

**Reply by tactilis** [reply]
The behaviour is the same in the Payees field

**Reply by tactilis** [reply]
Also, you are advised to update to v1.5.17, which has just been released. It contains some fixes to the operation of these fields

**Reply by ghost** [reply]
i've already installed it ! many thx

**Comment by vomikan** [comment]
You need to start writing the first letters of the desired category. Then it becomes possible to stretch the window.
![image](https://user-images.githubusercontent.com/6836805/179562335-92bf3247-5872-47fa-a094-9afff1144a55.png)
